src/global/UserContext/UserContext.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useState, Dispatch, SetStateAction, useEffect } from 'react';
+import { gqlUser } from 'global/graphqls/schema';
+import { GET_USER } from 'global/graphqls/User';
+import { useQuery } from '@apollo/react-hooks';
+import { createCtx } from 'helpers/Context';
+
+interface UserContext {
+  user: gqlUser | null;
+  setUser: Dispatch<SetStateAction<gqlUser | null>>;
+}
+
+const [useUserContext, UserContext] = createCtx<UserContext>();
+
+const UserProvider: FC = ({ children }) => {
+  const [user, setUser] = useState<gqlUser | null>(null);
+  const { data } = useQuery<{ user: gqlUser }>(GET_USER, {
+    variables: { condition: { uuid: localStorage.getItem('USER_UUID') } },
```

Comment:
Please remove UUID from localStorage since You got the current user context from BE based on accessToken